ci-operator/REHEARSALS.md

```diff
@@ -0,0 +1,39 @@
+CI job rehearsals
+=================
+
+**This feature is not fully deployed to production yet**
+
+Pull requests to this repository that contain changes affecting Prow job setup
+trigger a so-called "rehearsal". Jobs that would be affected by a PR are
+executed as if run against a target component repository after job changes
+would be merged. This provides job config authors early feedback about how job
+config changes impact CI setup for a given repo.
+
+## How rehearsal works
+
+All pull requests trigger a `ci/prow/pj-rehearse` check, which detects the jobs
+affected by the proposed change. For each job selected for rehearsal (for
+various reasons, we are not able to rehearse all jobs), new Prowjob is
```

Comment:
You already mention this parenthesis below

Reply:
That's intentional. Not everyone reads the whole thing. Here I mention it as a fact, and below I explain details about it.